modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -474,6 +479,76 @@ public int getOneToManyRelatedObjectEntriesCount(
 		return objectEntryPersistence.dslQueryCount(dslQuery);
 	}
 
+	@Override
+	public List<BaseModel<?>> getSystemUnrelatedObjectEntries(
+			long companyId, ObjectDefinition objectDefinition,
+			long objectFieldId,
+			SystemObjectDefinitionMetadata systemObjectDefinitionMetadata)
+		throws PortalException {
+
+		if (!objectDefinition.isSystem()) {
+			return new ArrayList<>();
+		}
+
+		Table systemObjectDefinitionMetadataTable =
+			systemObjectDefinitionMetadata.getTable();
+
+		Column<?, Long> companyIdColumn =
+			(Column<?, Long>)systemObjectDefinitionMetadataTable.getColumn(
+				"companyId");
+
+		PersistedModelLocalService persistedModelLocalService =
+			_persistedModelLocalServiceRegistry.getPersistedModelLocalService(
+				objectDefinition.getClassName());
+
+		List<BaseModel<?>> baseModels = persistedModelLocalService.dslQuery(
+			DSLQueryFactoryUtil.select(
+				systemObjectDefinitionMetadataTable
```

Comment:
Hey @marcelabc, you can leave this parameter empty if you're trying to bring all columns from a table.